GOVERNANCE.md

```diff
@@ -0,0 +1,68 @@
+# CRI-O contributor tiers
+
+* CRI-O has three tiers of code contributors: maintainers, emeritus maintainers, and community members.
+* Anyone can (and is encouraged to) be a community member, whose responsibilities are up to the contributor (submitting PRs, reviewing PRs).
+* Maintainers are the set of people that have final say over the direction of the project.
+
+# Process for becoming a maintainer
+
+* Express an interest to the current maintainers that you wish to become one.
+* Becoming a maintainer indicates you are going to be spending substantial (>25%) development time on CRI-O for the foreseeable future.
+* You should have domain expertise and be proficient in Go.
+* We will expect you to start contributing PRs under the guidance of current maintainers.
+* To start, we may ask you to take some issues from our backlog.
+* As you gain experience with the code base and our standards, we will ask you to do code reviews for incoming PRs (i.e., all maintainers are expected to shoulder a proportional share of community reviews).
+* After a period of approximately 2-3 months of working together and evaluating your contributions to the project, the existing maintainers will confer and decide whether to grant maintainer status or not. We make no guarantees on the length of time this will take, but 2-3 months is the approximate goal.
+
+## Maintainer responsibilities
+
+* Monitor our channel (#crio) on the Kubernetes Slack instance and our IRC channel (#cri-o) on Freenode (delayed response is perfectly acceptable)
+* Triage GitHub issues and perform pull request reviews for other maintainers and the community. The areas of specialization listed in [OWNERS.md](OWNERS.md) can be used to help with routing an issue/question to the right person.
+* Triage CI issues - file issues for known flaky CI runs or bugs, and either fix or find someone to fix any breakages.
+* During GitHub issue triage, apply all applicable [labels](https://github.com/cri-o/cri-o/labels) to each new issue. Labels are extremely useful for future issue follow up. Which labels to apply is somewhat subjective so just use your best judgment. A few of the most important labels that are not self explanatory are:
+  * **good first issue**: Mark any issue that can reasonably be accomplished by a new contributor with this label.
+  * **help wanted**: Unless it is immediately obvious that someone is going to work on an issue (and if so assign it), mark it help wanted.
+  * **CRI change**: If sufficiently fixing the issue involves a change to the CRI, this label is given. Often, maintainers won't have time to make such a change, but the change could be seen as good and as such the issue isn't closed.
```

Comment:
Do you have a 'kind/feature' label and do you want to also highlight that?

Reply:
that's more for PRs, for issues we have `area/vm` but I don't think it's common enough to warrant adding right now